util/network-devp2p/src/session.rs

```diff
@@ -372,7 +376,7 @@ impl Session {
 			},
 			PACKET_GET_PEERS => Ok(SessionData::None), //TODO;
 			PACKET_PEERS => Ok(SessionData::None),
-			PACKET_USER ... PACKET_LAST => {
+			PACKET_USER ..= PACKET_LAST => {
```

Comment:
`range_pattern` is not deprecated on stable yet right or have I missed something?

However, good catch all good from my side